scripts/creation-portal-migration/template/migrate/common.js

```diff
@@ -0,0 +1,34 @@
+const { isEmpty } = require("lodash");
+
+/**
+ * Map section data
+ * @method
+ * @name getCriteriaData
+ * @param {Object} criteria - criteria
+ * @param {String} type  - observation or survey
+ * @param {Object} question - criteria
+ * @returns  - returns the mapped section
+ **/
+const getCriteriaData = (criteria, type, question = {}) => {
+  if (isEmpty(question)) {
+    return {
+      code: criteria?.externalId,
+      name: criteria?.name,
+      description: criteria?.description,
+      mimeType: "application/vnd.sunbird.questionset",
+      primaryCategory: type,
+    };
+  } else {
+    return {
+      code: question?.externalId,
```

Comment:
@jayesh9747 in the readme file mention when to rerun the script, like u said if data is in logger need to rerun it right ?

Reply:
can you explain in detail , i dont understand what you say?

Reply:
In readme you mention how to run the script right ? , But how the people will understand whento rerun the script. You said when something failed it will add to error log right. So please add that point also